R/n4BiasFieldCorrection.R

```diff
@@ -44,7 +46,8 @@ outimg<-antsImageClone(img)
        c = N4_CONVERGENCE_1,
        b = N4_BSPLINE_PARAMS,
        x = mask,
-       o = outimg)
+       o = outimg,
+       v = v)
```

Comment:
should this `v = v` actually be `v = as.numeric( verbose )`  in addition the default value for verbose should FALSE